src/decorators/debounce.ts

```diff
@@ -1,6 +1,12 @@
-import lodashDebounce, {DebounceSettings} from 'lodash/debounce';
+import debounce from 'lodash/debounce';
 
-export default function debounce(wait = 500, options?: DebounceSettings) {
+export interface DebounceSettings {
+    leading?: boolean,
+    maxWait?: number,
+    trailing?: boolean,
+}
+
+export default function debounceDecorator(wait = 500, options?: DebounceSettings) {
```

Comment:
You're getting flagged by the `match-default-export-name` tslint rule. 
It's turned off in neutron explicitly. I think we've gauged enough feedback from it to turn it off in our tslint config as well. I'll do that later today. For now, I think you should just ignore the `match-default-export-name` rule inline